packages/blockchain/CHANGELOG.md

```diff
@@ -6,6 +6,94 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+
+Along with the breaking releases we have therefore deactivated both of these options and you might therefore need to adapt some import statements accordingly. Note that you still can activate these options in your bundle and/or transpilation pipeline (but now you also have the option *not* to, which you didn't have before).
+
+### BigInt-Related and other API Changes
+
+Different methods in the Blockchain library have been renamed for clarity or have a slightly different API due to the BigInt introduction, see PRs [#1822](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1822) and [#1877](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1877):
+
+- `getLatestHeader()` -> `getCanonicalHeadHeader()`
+- `getLatestBlock()` -> `getCanonicalHeadBlock()`
+- `iterator(name: string, onBlock: OnBlock): Promise<void | number>` -> `iterator(name: string, onBlock: OnBlock): Promise<number>`
+
+The following getters and/or methods have been removed:
+
+- `get meta()`
+- `getHead()` (use `getIteratorHead()` instead)
+- `setHead()` (use `setIteratorHead()` instead)
+
+### Consensus Encapsulation
+
+Consensus-related functionality in the `Blockchain` package has been reworked and taken out of the main `Blockchain` class, see PR [#1756](https://github.com/ethereumjs/ethereumjs-monorepo/issues/1756).
+
+There is now a dedicated consensus class for each type of supported consensus, `Ethash`, `Clique` and `Casper` (PoS, this one is rather the do-nothing part of `Casper` and letting the respective consensus/beacon client do the hard work! 🙂). Each consensus class adheres to a common interface `Consensus` implementing the following five methods in a consensus-specific way:
+
+- `genesisInit(genesisBlock: Block): Promise<void>`
+- `setup(): Promise<void>`
+- `validateConsensus(block: Block): Promise<void>`
+- `validateDifficulty(header: BlockHeader): Promise<void>`
+- `newBlock(block: Block, commonAncestor?: BlockHeader, ancientHeaders?: BlockHeader[]): Promise<void>`
+
+There is now a new `Blockchain` option `consensus`. This makes it very easy to pass in a modified version of an existing consensus implementation or do something totally different and write an own consensus class with consensus rules to follow.
+
+### Added Genesis Functionality
+
+PRs [#1916](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1916) and - as some follow-up work - [#1924](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1924) rework the genesis code throughout the EthereumJS library stack, with benefits on the bundle size of the lower level libraries (like `Block` or `Transaction`).
+
+In return the `Blockchain` class has gotten now respective responsibilities. Genesis state and block functionality previously in the `@ethereumjs/common` class has been integrated here, see PR [#1916](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1916).
```

Comment:
```suggestion
In return the `Blockchain` class has gotten now respective responsibilities. Genesis state and block functionality previously in the `@ethereumjs/common` class have been integrated here, see PR [#1916](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1916).
```

Reply:
I'm not sure I understand this sentence:
'In return the `Blockchain` class has gotten now respective responsibilities.'
Would it be possible to clarify?

Reply:
Will add some clarification.

Have submitted the other change suggestions, thanks for the review! 🙂